l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -584,7 +586,8 @@ def setDettaglioLinee(self, invoice, body):
             DettaglioLinea = DettaglioLineeType(
                 NumeroLinea=str(line_no),
                 Descrizione=line.name,
-                PrezzoUnitario='%.2f' % prezzo_unitario,
+                PrezzoUnitario=eval(
```

Comment:
As far as I know `eval` is often a bad choice, could you please format the string in some other way?

Reply:
@SimoRubi done, thanks